CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Change log
 
+## 2019-7-8 - Release - 2.0.0
+### Summary
```

Comment:
FWIW, empty sections seem odd. Also, for this to render correctly everywhere there should be a line between the h2 and h3 components